lib/reporting/drop_off_report.rb

```diff
@@ -370,10 +385,12 @@ def to_csvs
 
     # @return [Float]
     def percent(numerator:, denominator:)
-      (numerator.to_f / denominator.to_f)
+      result = (numerator.to_f / denominator.to_f)
+      result.nan? ? 0 : result
```

Comment:
Strictly speaking this is new behavior not defined in the ticket. I'm tempted to remove it, which would fall back on the relevant columns in the report containing "NaN", which is how Ruby evaluates `Float::NAN.to_s`

Reply:
nan comes up in other reports, we might want to consider adding a NaN check in the renderer itself?

https://github.com/18F/identity-idp/blob/604aa3ce55a6f1c362772877e6946f78abfe2f01/app/views/report_mailer/tables_report.html.erb#L23-L27

Or maybe the `EmailableReport` constructor can convert NaNs to zeroes so that CSVs benefit from the fix as well

Reply:
This should only be NaN if the denominator is zero, correct? (Or if we're returning something other than a number for either arg)
So returning `0` would be incorrect.

Reply:
It would be technically incorrect, yes. That said, what's technically correct and what clearly communicates something intuitively understandable to the users are regularly not the same thing. My thinking is, if someone receiving this report knows it ought to be NaN when zero appears in this cell, then they likely know enough to not be surprised as to why a zero might appear instead. And If they are confused by NaN and don't understand what it could mean, then putting a zero in instead stands a good chance of helping them not be confused.

All that said, I'm rolling this change back in the interest of getting something out that doesn't include a half-assed edit

Reply:
The specific reason I mention it is that if someone is plugging these numbers into a spreadsheet somewhere, there's a risk that zeroes change their averages and it may not be readily apparent to them that anything is off.

Reply:
Oh, right. Now I remember, in trying to roll it back, one reason I made this change in the first place. As per the IEEE spec, NaN == NaN must return false, so it's hard to get the tests to pass even when they're doing the right thing.

Reply:
Wow, the workarounds for this are just wild: `var != var` will return `true` when `var` is `NaN` for that reason. I